Fix typos in LowLevelInfo (#1914)
Haivision/srt: docs/LowLevelInfo.md

## Introduction

> This document will contain loose information for various topics referring to
> the SRT source code describing some cross-source analysis that would be

Reviewer: ```suggestion
This document contains information on various topics related to
the SRT source code, including descriptions of some cross-source analysis that would be
```

> non-obvious for a source code reviewer. It's not a complete documentation of

Reviewer 2: Not sure about this change: "unobvious" -> "non-obvious". @stevomatthews what do you think?

Reviewer: "that would not be obvious for a source code reviewer" is the safe way to go!  ;-)

Reviewer 2: @quink-black Please also apply this suggestion.

Author: Done. It's amazing I can learning English from this project : P
